KHPCPatchManager.cs

```diff
@@ -137,50 +170,17 @@ class KHPCPatchManager{
 						var egs = new OpenKh.Command.IdxImg.Program.EpicGamesAssets.PatchCommand();
 						egs.Execute(epicPkgBackupFile, patchFolder, epicFolder);
 					}else{
-						Console.WriteLine($"Could not find {kh1files[i]} n/or any patch for it.");
-					}
-				}
-				Directory.Delete(timestamp, true);
-				Console.WriteLine("Done!");
-			}else if(kh2pcpatchFile != null){
-				Console.WriteLine("Applying KH2 patch...");
-				string epicFolder = null;
-				while(!Directory.Exists(epicFolder)){
-					Console.WriteLine("Please drag your \"en\" folder (the one that contains kh2_first, kh2_second, etc.) located under \"Kingdom Hearts HD 1 5 and 2 5 ReMIX/Image/\" here:");
-					epicFolder = Console.ReadLine().Trim('"');
-				}
-				Console.WriteLine("Extracting patch...");
-				string timestamp = kh2pcpatchFile + "_" + DateTime.Now.ToString("dd_MM_yyyy_HH_mm_ss_ms");
-				ZipFile zip = ZipFile.Read(kh2pcpatchFile);
-				Directory.CreateDirectory(timestamp);
-				string epicBackup = Path.Combine(epicFolder, "backup");
-				Directory.CreateDirectory(epicBackup);
-				zip.ExtractAll(timestamp, ExtractExistingFileAction.OverwriteSilently);
-				for(int i=0;i<kh2files.Length;i++){
-					string epicFile = Path.Combine(epicFolder, kh2files[i] + ".pkg");
-					string epicHedFile = Path.Combine(epicFolder, kh2files[i] + ".hed");
-					string patchFolder = Path.Combine(timestamp, kh2files[i]);
-					string epicPkgBackupFile = Path.Combine(epicBackup, kh2files[i] + ".pkg");
-					string epicHedBackupFile = Path.Combine(epicBackup, kh2files[i] + ".hed");
-					if(Directory.Exists(patchFolder) && File.Exists(epicFile)){
-						if(File.Exists(epicPkgBackupFile)) File.Delete(epicPkgBackupFile);
-						File.Move(epicFile, epicPkgBackupFile);
-						if(File.Exists(epicHedBackupFile)) File.Delete(epicHedBackupFile);
-						File.Move(epicHedFile, epicHedBackupFile);
-						Console.WriteLine($"Patching {epicFile}...");
-						var egs = new OpenKh.Command.IdxImg.Program.EpicGamesAssets.PatchCommand();
-						egs.Execute(epicPkgBackupFile, patchFolder, epicFolder);
-					}else{
-						Console.WriteLine($"Could not find {kh2files[i]} n/or any patch for it.");
+						Console.WriteLine($"Could not find {khFiles[i]} n/or any patch for it.");
 					}
 				}
 				Directory.Delete(timestamp, true);
 				Console.WriteLine("Done!");
+
 			}else{
 				Console.WriteLine("- Drop a .hed file to unpack the associated .pkg file");
 				Console.WriteLine("- Drop a .pkg file and its unpacked folder to patch it");
-				Console.WriteLine("- Drop a folder(s) (extracted .pkg format) to create a kh2pcpatch or a kh2pcpatch");
-				Console.WriteLine("- Drop a kh1pcpatch or a kh2pcpatch to patch your .pkgs");
+				Console.WriteLine("- Drop a folder(s) (extracted .pkg format) to create a kh1pcpatch, kh2pcpatch or a bbspatch");
+				Console.WriteLine("- Drop a kh1pcpatch, kh2pcpatch or a bbspatch to patch your .pkgs");
```

Comment:
"bbspatch" should be changed to "bbs**pc**patch" to be consistent with other games' patch format, as well as the file extension you've already set before.
Probably just a typo or you got distracted :p?

Reply:
Whoopsie, that's a nice typo :P